### DISCUSSION TITLE
No valid templates were found (latest version, latest templates version)

### DISCUSSION THREAD

**Discussion by wyv3rnnn** [projectdiscovery/nuclei#1551] [Q&A] [answered]
### Nuclei version:

2.5.9

### Nuclei Templates version:
8.8.3

### Current Behavior:

Currently when you try to do a scan for web technologies it will give you the error "No valid templates were found" even though entire template folder is filled to the brim with templates that were working in previous versions (see screenshot below). 
<img width="1792" alt="Screen Shot 2022-01-27 at 7 02 09 PM" src="https://user-images.githubusercontent.com/77228062/151463407-a56b77ce-832f-4b85-9f43-e5f138556795.png">

### Expected Behavior:

When running the command, it should just work and not complain about "no valid templates". 

Steps To Reproduce Behavior:

1. Run 'nuclei -t nuclei-templates/technologies/ -u https://example.com'
2. See error...
<img width="668" alt="Screen Shot 2022-01-27 at 6 57 25 PM" src="https://user-images.githubusercontent.com/77228062/151462967-df0ebfb7-0920-4c95-a69c-86e74ef36986.png">

Any help would be much appreciated!

Thanks,
WYV3RN

**Comment by geeknik** [comment]
I've tested with both 2.5.9 and 2.6.0-dev and I can't reproduce this issue.

**Comment by wyv3rnnn** [comment]
???? That's exactly what I ran...
<img width="1214" alt="Screen Shot 2022-01-27 at 7 26 14 PM" src="https://user-images.githubusercontent.com/77228062/151465640-b79444c3-e47c-4b31-93cb-bb129eef50c1.png">

**Comment by ehsandeep** [comment]
@wyv3rnnn can you include full screenshot including cmd as well? or share the output running the same cmd with `-v` flag.

**Comment by forgedhallpass** [comment]
@wyv3rnnn could you also make sure that the folder you've listed above coincides with the path configured in the `nuclei-templates-directory` under the `~/.config/nuclei/.templates-config.json` file, and the `update-directory` in the `~/.config/nuclei/config.yaml` file points to the same folder (both should be absolute paths)?

**Comment by wyv3rnnn** [comment, marked as answer]
So I just ran the same command today and it auto-updated the "nuclei-templates" version from 8.8.3 to 8.8.4 and now everything is working again? I'm not sure what the problem was now... 

<img width="1605" alt="Screen Shot 2022-01-28 at 11 02 12 AM" src="https://user-images.githubusercontent.com/77228062/151580223-62a810b9-ec1e-4336-9101-1506846f09cf.png">
